.github/workflows/test.yml

```diff
@@ -200,6 +200,27 @@ jobs:
             **/target/surefire-reports/
             **/hs_err_*.log
           retention-days: 7
+  java-randomized-tests:
+    name: Java randomized tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'maven'
+      - run: mvn -T1C -B -D skipChecks -D skipTests install
```

Comment:
I don't think this line is necessary. The next line should compile the code anyway. Also the install step is not really necessary, afaik

Reply:
We definitely need to install build-tools first. I'm not sure about other modules, I can give it a try.

Reply:
Ah, I keep forgetting about that bit :-)